node-require/calculate.js

```diff
@@ -0,0 +1,24 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+const calculation = (x, test, y) => {
+  x = Number(process.argv[process.argv.length - 3]);
+  test = process.argv[process.argv.length - 2];
+  y = Number(process.argv[process.argv.length - 1]);
```

Comment:
If you plan on defining variables inside a function, don't include them in the function's parameter list.